pkg/grid/src/nav/Help.tsx

```diff
@@ -9,12 +10,65 @@ export const Help = () => {
   }, []);
 
   return (
-    <div className="p-4 md:p-8 space-y-8">
-      <h2 className="h4 text-gray-500">Recent Apps</h2>
-      <div className="min-h-[150px] rounded-xl bg-gray-100" />
-      <hr className="-mx-4 md:-mx-8" />
-      <h2 className="h4 text-gray-500">Recent Developers</h2>
-      <div className="min-h-[150px] rounded-xl bg-gray-100" />
+    <div className="flex flex-col items-center px-4 py-8 md:px-8 md:py-16 space-y-8 md:space-y-16">
+      <img className="w-52 h-auto" src={helpAndSupport} alt="" />
+      <div className="h4 space-y-4 md:space-y-8">
+        <div>
+          <h2>User Help Docs</h2>
+          <a
+            className="text-blue-400"
+            href="mailto:support@tlon.io"
+            target="_blank"
+            rel="noreferrer"
+          >
+            tlon.io/help
+          </a>
+        </div>
+        <div>
+          <h2>For general help, ask the community</h2>
+          <a
+            className="text-blue-400"
+            href="web+urbitgraph://group/~bitbet-bolbel/urbit-community"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Join Urbit Community
+          </a>
+        </div>
+        <div>
+          <h2>For general help, ask the community</h2>
+          <p>
+            <a
+              className="text-blue-400"
+              href="web+urbitgraph://group/~bitpyx-dildus/tlon-support"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Join Tlon Support
+            </a>
+            <span className="mx-2 text-black">or email</span>
```

Comment:
Cc: @urcades tlon support is only for hosting customers, yes?

Reply:
Hmm if it is, how would we be able to differentiate people here to show/hide?